src/librustc_mir/borrow_check/nll/type_check/mod.rs

```diff
@@ -919,6 +944,112 @@ impl<'a, 'gcx, 'tcx> TypeChecker<'a, 'gcx, 'tcx> {
         )
     }
 
+    fn eq_opaque_type_and_type(
+        &mut self,
+        revealed_ty: Ty<'tcx>,
+        anon_ty: Ty<'tcx>,
```

Comment:
Similar here.

Reply:
I think `anon_ty` as a parameter name fits in with the naming scheme elsewhere, but not sure...

Reply:
That naming scheme probably got missed by the anon -> opaque mass rename.

Reply:
Probably yep.